Rakefile

```diff
@@ -22,6 +26,9 @@ file PARSER_CACHE_DIR => [BUILDDIR] do
 end
 
 file "#{BUILDDIR}/modules" do |t|
+  # Append Foreman's Puppetfile to ours, so we use the right version of the puppet modules for the cache
+  File.open('Puppetfile', 'a') { |f| f.write(open("https://raw.githubusercontent.com/theforeman/foreman-installer/#{FOREMAN_BRANCH}/Puppetfile").read) }
```

Comment:
The foreman modules already get installed because they're dependencies, but it's wrong the versions because they'll pulled from puppet forge